[Conflict] Overloaded delay function in FastLED conflicts with IRremote

### Board / MCU

ESP32-S3

### LED Chipset

WS2812B / NeoPixel

### FastLED Version

master#a843234

### Arduino / PlatformIO version

PlatformIO 6.1.19

### Operating System

Windows

### Description

If you attempt to compile a sketch that uses both FastLED `master#a843234` and [IRremote](https://github.com/Arduino-IRremote/Arduino-IRremote/releases/tag/v4.7.1), the compile will fail due to a conflict in overloading the basic Arduino `delay` function.

PIO lib_deps:
```
https://github.com/FastLED/FastLED.git#a843234 ; https://github.com/FastLED/FastLED
arduino-irremote/IRremote@^4.7.1 ; https://github.com/Arduino-IRremote/Arduino-IRremote
```
Compilation error:
```
In file included from .pio/libdeps/esp32s3/IRremote/src/IRremote.hpp:272,
                 from C:/Users/user/git/N1/haslab-proton-pack/source/SharedLib/InfraredManager/include/InfraredManager.h:114,
                 from C:/Users/user/git/N1/haslab-proton-pack/source/SharedLib/InfraredManager/include/InfraredManager.hpp:26,
                 from src/main.cpp:87:
.pio/libdeps/esp32s3/IRremote/src/IRSend.hpp: In member function 'void IRsend::sendRaw(const uint16_t*, uint_fast16_t, uint_fast8_t, uint_fast16_t, int_fast8_t)':
.pio/libdeps/esp32s3/IRremote/src/IRSend.hpp:443:22: error: call of overloaded 'delay(long unsigned int)' is ambiguous
  443 |                 delay(aRepeatPeriodMillis - tCurrentFrameDurationMillis);
      |                 ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from C:/platform-io/packages/framework-arduinoespressif32/cores/esp32/Arduino.h:44,
                 from src/main.cpp:29:
C:/platform-io/packages/framework-arduinoespressif32/cores/esp32/esp32-hal.h:142:6: note: candidate: 'void delay(uint32_t)'
  142 | void delay(uint32_t);
      |      ^~~~~
In file included from .pio/libdeps/esp32s3/FastLED/src/fastled_delay.h:8,
                 from .pio/libdeps/esp32s3/FastLED/src/fl/fastled.h:89,
                 from .pio/libdeps/esp32s3/FastLED/src/FastLED.h:158,
                 from src/main.cpp:55:
.pio/libdeps/esp32s3/FastLED/src/fl/system/delay.h:87:6: note: candidate: 'void fl::delay(u32, bool)'
   87 | void delay(u32 ms, bool run_async = true) FL_NOEXCEPT;
      |      ^~~~~
.pio/libdeps/esp32s3/IRremote/src/IRSend.hpp: In member function 'void IRsend::sendRaw(const uint8_t*, uint_fast16_t, uint_fast8_t, uint_fast16_t, int_fast8_t)':
.pio/libdeps/esp32s3/IRremote/src/IRSend.hpp:483:22: error: call of overloaded 'delay(long unsigned int)' is ambiguous
  483 |                 delay(aRepeatPeriodMillis - tCurrentFrameDurationMillis);
      |                 ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/platform-io/packages/framework-arduinoespressif32/cores/esp32/esp32-hal.h:142:6: note: candidate: 'void delay(uint32_t)'
  142 | void delay(uint32_t);
      |      ^~~~~
.pio/libdeps/esp32s3/FastLED/src/fl/system/delay.h:87:6: note: candidate: 'void fl::delay(u32, bool)'
   87 | void delay(u32 ms, bool run_async = true) FL_NOEXCEPT;
      |      ^~~~~
.pio/libdeps/esp32s3/IRremote/src/IRSend.hpp: In member function 'void IRsend::sendRaw_P(const uint16_t*, uint_fast16_t, uint_fast8_t, uint_fast16_t, int_fast8_t)':
.pio/libdeps/esp32s3/IRremote/src/IRSend.hpp:546:22: error: call of overloaded 'delay(long unsigned int)' is ambiguous
  546 |                 delay(aRepeatPeriodMillis - tCurrentFrameDurationMillis);
      |                 ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/platform-io/packages/framework-arduinoespressif32/cores/esp32/esp32-hal.h:142:6: note: candidate: 'void delay(uint32_t)'
  142 | void delay(uint32_t);
      |      ^~~~~
.pio/libdeps/esp32s3/FastLED/src/fl/system/delay.h:87:6: note: candidate: 'void fl::delay(u32, bool)'
   87 | void delay(u32 ms, bool run_async = true) FL_NOEXCEPT;
      |      ^~~~~
.pio/libdeps/esp32s3/IRremote/src/IRSend.hpp: In member function 'void IRsend::sendRaw_P(const uint8_t*, uint_fast16_t, uint_fast8_t, uint_fast16_t, int_fast8_t)':
.pio/libdeps/esp32s3/IRremote/src/IRSend.hpp:594:22: error: call of overloaded 'delay(long unsigned int)' is ambiguous
  594 |                 delay(aRepeatPeriodMillis - tCurrentFrameDurationMillis);
      |                 ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/platform-io/packages/framework-arduinoespressif32/cores/esp32/esp32-hal.h:142:6: note: candidate: 'void delay(uint32_t)'
  142 | void delay(uint32_t);
      |      ^~~~~
.pio/libdeps/esp32s3/FastLED/src/fl/system/delay.h:87:6: note: candidate: 'void fl::delay(u32, bool)'
   87 | void delay(u32 ms, bool run_async = true) FL_NOEXCEPT;
      |      ^~~~~
.pio/libdeps/esp32s3/IRremote/src/IRSend.hpp: In member function 'void IRsend::sendPulseDistanceWidth(uint_fast8_t, uint16_t, uint16_t, uint16_t, uint16_t, uint16_t, uint16_t, IRDecodedRawDataType, uint_fast8_t, uint8_t, uint16_t, int_fast8_t, void (*)())':
.pio/libdeps/esp32s3/IRremote/src/IRSend.hpp:733:22: error: call of overloaded 'delay(long unsigned int)' is ambiguous
  733 |                 delay(aRepeatPeriodMillis - tFrameDurationMillis);
      |                 ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/platform-io/packages/framework-arduinoespressif32/cores/esp32/esp32-hal.h:142:6: note: candidate: 'void delay(uint32_t)'
  142 | void delay(uint32_t);
      |      ^~~~~
.pio/libdeps/esp32s3/FastLED/src/fl/system/delay.h:87:6: note: candidate: 'void fl::delay(u32, bool)'
   87 | void delay(u32 ms, bool run_async = true) FL_NOEXCEPT;
      |      ^~~~~
.pio/libdeps/esp32s3/IRremote/src/IRSend.hpp: In member function 'void IRsend::sendPulseDistanceWidthFromArray(uint_fast8_t, uint16_t, uint16_t, uint16_t, uint16_t, uint16_t, uint16_t, IRDecodedRawDataType*, uint16_t, uint8_t, uint16_t, int_fast8_t)':
.pio/libdeps/esp32s3/IRremote/src/IRSend.hpp:798:22: error: call of overloaded 'delay(long unsigned int)' is ambiguous
  798 |                 delay(aRepeatPeriodMillis - tFrameDurationMillis);
      |                 ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/platform-io/packages/framework-arduinoespressif32/cores/esp32/esp32-hal.h:142:6: note: candidate: 'void delay(uint32_t)'
  142 | void delay(uint32_t);
      |      ^~~~~
.pio/libdeps/esp32s3/FastLED/src/fl/system/delay.h:87:6: note: candidate: 'void fl::delay(u32, bool)'
   87 | void delay(u32 ms, bool run_async = true) FL_NOEXCEPT;
      |      ^~~~~
.pio/libdeps/esp32s3/IRremote/src/IRSend.hpp: In member function 'void IRsend::sendPulseDistanceWidthFromPGMArray(uint_fast8_t, uint16_t, uint16_t, uint16_t, uint16_t, uint16_t, uint16_t, const IRDecodedRawDataType*, uint16_t, uint8_t, uint16_t, int_fast8_t)':
.pio/libdeps/esp32s3/IRremote/src/IRSend.hpp:869:22: error: call of overloaded 'delay(long unsigned int)' is ambiguous
  869 |                 delay(aRepeatPeriodMillis - tFrameDurationMillis);
      |                 ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/platform-io/packages/framework-arduinoespressif32/cores/esp32/esp32-hal.h:142:6: note: candidate: 'void delay(uint32_t)'
  142 | void delay(uint32_t);
      |      ^~~~~
.pio/libdeps/esp32s3/FastLED/src/fl/system/delay.h:87:6: note: candidate: 'void fl::delay(u32, bool)'
   87 | void delay(u32 ms, bool run_async = true) FL_NOEXCEPT;
      |      ^~~~~
.pio/libdeps/esp32s3/IRremote/src/IRSend.hpp: In member function 'void IRsend::sendPulseDistanceWidth(PulseDistanceWidthProtocolConstants*, IRDecodedRawDataType, uint_fast8_t, int_fast8_t)':
.pio/libdeps/esp32s3/IRremote/src/IRSend.hpp:967:22: error: call of overloaded 'delay(long unsigned int)' is ambiguous
  967 |                 delay(aProtocolConstants->RepeatPeriodMillis - tCurrentFrameDurationMillis);
      |                 ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/platform-io/packages/framework-arduinoespressif32/cores/esp32/esp32-hal.h:142:6: note: candidate: 'void delay(uint32_t)'
  142 | void delay(uint32_t);
      |      ^~~~~
.pio/libdeps/esp32s3/FastLED/src/fl/system/delay.h:87:6: note: candidate: 'void fl::delay(u32, bool)'
   87 | void delay(u32 ms, bool run_async = true) FL_NOEXCEPT;
      |      ^~~~~
.pio/libdeps/esp32s3/IRremote/src/IRSend.hpp: In member function 'void IRsend::sendPulseDistanceWidthFromArray(PulseDistanceWidthProtocolConstants*, IRDecodedRawDataType*, uint16_t, int_fast8_t)':
.pio/libdeps/esp32s3/IRremote/src/IRSend.hpp:1050:22: error: call of overloaded 'delay(long unsigned int)' is ambiguous
 1050 |                 delay(aProtocolConstants->RepeatPeriodMillis - tFrameDurationMillis);
      |                 ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/platform-io/packages/framework-arduinoespressif32/cores/esp32/esp32-hal.h:142:6: note: candidate: 'void delay(uint32_t)'
  142 | void delay(uint32_t);
      |      ^~~~~
.pio/libdeps/esp32s3/FastLED/src/fl/system/delay.h:87:6: note: candidate: 'void fl::delay(u32, bool)'
   87 | void delay(u32 ms, bool run_async = true) FL_NOEXCEPT;
      |      ^~~~~
.pio/libdeps/esp32s3/IRremote/src/IRSend.hpp: In member function 'void IRsend::sendPulseDistanceWidthFromPGMArray(PulseDistanceWidthProtocolConstants*, const IRDecodedRawDataType*, uint16_t, int_fast8_t)':
.pio/libdeps/esp32s3/IRremote/src/IRSend.hpp:1129:22: error: call of overloaded 'delay(long unsigned int)' is ambiguous
 1129 |                 delay(aProtocolConstants->RepeatPeriodMillis - tFrameDurationMillis);
      |                 ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/platform-io/packages/framework-arduinoespressif32/cores/esp32/esp32-hal.h:142:6: note: candidate: 'void delay(uint32_t)'
  142 | void delay(uint32_t);
      |      ^~~~~
.pio/libdeps/esp32s3/FastLED/src/fl/system/delay.h:87:6: note: candidate: 'void fl::delay(u32, bool)'
   87 | void delay(u32 ms, bool run_async = true) FL_NOEXCEPT;
      |      ^~~~~
.pio/libdeps/esp32s3/IRremote/src/IRSend.hpp: In static member function 'static void IRsend::customDelayMicroseconds(long unsigned int)':
.pio/libdeps/esp32s3/IRremote/src/IRSend.hpp:1473:14: error: call of overloaded 'delay(long unsigned int)' is ambiguous
 1473 |         delay(aMicroseconds / 1000UL);  // Delay for as many whole milliseconds as we can.
      |         ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~
C:/platform-io/packages/framework-arduinoespressif32/cores/esp32/esp32-hal.h:142:6: note: candidate: 'void delay(uint32_t)'
  142 | void delay(uint32_t);
      |      ^~~~~
.pio/libdeps/esp32s3/FastLED/src/fl/system/delay.h:87:6: note: candidate: 'void fl::delay(u32, bool)'
   87 | void delay(u32 ms, bool run_async = true) FL_NOEXCEPT;
      |      ^~~~~
In file included from .pio/libdeps/esp32s3/IRremote/src/IRremote.hpp:287:
.pio/libdeps/esp32s3/IRremote/src/ir_RC5_RC6.hpp: In member function 'void IRsend::sendRC5Marantz(uint8_t, uint8_t, int_fast8_t, uint8_t, bool)':
.pio/libdeps/esp32s3/IRremote/src/ir_RC5_RC6.hpp:219:18: error: call of overloaded 'delay(long unsigned int)' is ambiguous
  219 |             delay(MARANTZ_REPEAT_DISTANCE / MICROS_IN_ONE_MILLI);
      |             ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/platform-io/packages/framework-arduinoespressif32/cores/esp32/esp32-hal.h:142:6: note: candidate: 'void delay(uint32_t)'
  142 | void delay(uint32_t);
      |      ^~~~~
.pio/libdeps/esp32s3/FastLED/src/fl/system/delay.h:87:6: note: candidate: 'void fl::delay(u32, bool)'
   87 | void delay(u32 ms, bool run_async = true) FL_NOEXCEPT;
      |      ^~~~~
.pio/libdeps/esp32s3/IRremote/src/ir_RC5_RC6.hpp: In member function 'void IRsend::sendRC5(uint8_t, uint8_t, int_fast8_t, bool)':
.pio/libdeps/esp32s3/IRremote/src/ir_RC5_RC6.hpp:268:18: error: call of overloaded 'delay(long unsigned int)' is ambiguous
  268 |             delay(RC5_REPEAT_DISTANCE / MICROS_IN_ONE_MILLI);
      |             ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/platform-io/packages/framework-arduinoespressif32/cores/esp32/esp32-hal.h:142:6: note: candidate: 'void delay(uint32_t)'
  142 | void delay(uint32_t);
      |      ^~~~~
.pio/libdeps/esp32s3/FastLED/src/fl/system/delay.h:87:6: note: candidate: 'void fl::delay(u32, bool)'
   87 | void delay(u32 ms, bool run_async = true) FL_NOEXCEPT;
      |      ^~~~~
Compiling .pio\build\esp32s3\lib33e\FastLED\fl\build\fl.details+.cpp.o
*** [.pio\build\esp32s3\src\main.cpp.o] Error 1
========================= [FAILED] Took 51.99 seconds =========================

Environment    Status    Duration
-------------  --------  ------------
esp32s3        FAILED    00:00:51.991
==================== 1 failed, 0 succeeded in 00:00:51.991 ====================
```

### Steps to reproduce

1. Have a sketch that uses both FastLED `master#a843234` and IRremote.
2. Attempt to compile the sketch.
3. Sketch fails to compile.

### Minimal sketch

```cpp
#include <Arduino.h>
#include <FastLED.h>

#define DISABLE_CODE_FOR_RECEIVER
#define NO_LED_FEEDBACK_CODE
#define EXCLUDE_UNIVERSAL_PROTOCOLS // Disable universal protocols; we only use NEC.
#define EXCLUDE_EXOTIC_PROTOCOLS // Disable exotic protocols; we only use NEC.
#define CARRIER_KHZ 38 // Defines the standard IR carrier frequency in kHz.
#ifndef IR_LED_PIN // Only set a default pin number if not already set.
  #define IR_LED_PIN 17 // Standard IR LED pin
#endif
#include <IRremote.hpp>

#define NUM_LEDS  55    // Number of NeoPixel leds
#define PIN       5     // Data pin for ledstrip

CRGB leds[NUM_LEDS];

void setup() 
{
  FastLED.addLeds<WS2812B, PIN, GRB>(leds, NUM_LEDS).setCorrection(TypicalLEDStrip);
  FastLED.setMaxRefreshRate(0); // Disable FastLED's blocking 2.5ms delay.
  FastLED.clear();       // turn all leds off
  FastLED.show();        // push update to LEDs
  IrSender.begin(IR_LED_PIN); // Initialize IR sender object.
}

void loop() 
{
  // nothing
}
```

### Checklist

- [x] I have searched existing issues for duplicates
- [x] I have tested with the latest FastLED release or git master
- [x] I can reproduce this with a minimal sketch

--- a/ci/lint_cpp/using_namespace_fl_in_examples_checker.py
+++ b/ci/lint_cpp/using_namespace_fl_in_examples_checker.py
@@ -2,8 +2,8 @@
 """Checker to ensure 'using namespace fl;' is not used in example files.
 
 This checker prevents namespace conflicts between fl:: and Arduino built-in functions.
-When FastLED.h does 'using fl::delay;' and an example also does 'using namespace fl;',
-it can create ambiguity with Arduino's delay() function.
+FastLED.h re-exports selected fl:: names, so examples should keep fl:: usage explicit
+instead of pulling the whole namespace into sketch scope.
 
 Examples should use:
 - Explicit qualification: fl::delay(), fl::XYMap, etc.

--- a/src/FastLED.h
+++ b/src/FastLED.h
@@ -1599,7 +1599,7 @@ using fl::UIDropdown;
 using fl::UIGroup;
 using fl::XYMap;
 using fl::round;  // Template version avoids conflicts with ::round
-using fl::delay;  // Template version coexists with Arduino's extern "C" delay()
+using fl::delay;  // Explicit fl::delay() stays available without hijacking bare Arduino delay()
 
 // Common fl:: type aliases for global namespace convenience
 template<typename T> using fl_vector = fl::vector<T>;

--- a/src/fl/stl/cstdio.cpp.hpp
+++ b/src/fl/stl/cstdio.cpp.hpp
@@ -10,8 +10,10 @@
 
 // Forward declare delay to avoid Arduino conflict
 namespace fl {
-    void delay(u32 ms, bool run_async);
     void delayMicroseconds(u32 us);
+    namespace detail {
+        void delay_impl(u32 ms, bool run_async);
+    }
 }
 
 // =============================================================================

--- a/src/fl/system/delay.cpp.hpp
+++ b/src/fl/system/delay.cpp.hpp
@@ -172,7 +172,9 @@ template<> void delaycycles<50>() { delaycycles<40>(); delaycycles<10>(); }
 // Millisecond and Microsecond delay implementations
 // ============================================================================
 
-void delay(u32 ms, bool run_async) {
+namespace detail {
+
+void delay_impl(u32 ms, bool run_async) {
 #if SKETCH_HAS_LARGE_MEMORY
   // Check if delay override is active (for fast testing with stub platform)
   // When override is active, skip async pumping and use platform delay directly
@@ -194,9 +196,11 @@ void delay(u32 ms, bool run_async) {
 #endif
 }
 
+}  // namespace detail
+
 void delayMillis(u32 ms) {
   // Legacy function - no async pumping (backward compatibility)
-  delay(ms, false);
+  detail::delay_impl(ms, false);
 }
 
 void delayMicroseconds(u32 us) {

--- a/src/fl/system/delay.h
+++ b/src/fl/system/delay.h
@@ -81,29 +81,23 @@ template<cycle_t CYCLES> inline void delaycycles_min1() FL_NOEXCEPT {
 // Millisecond and Microsecond delay wrappers
 // ============================================================================
 
-/// Delay for a given number of milliseconds with optional async task pumping
+namespace detail {
+
+/// Internal delay implementation used by the public fl::delay wrapper
 /// @param ms Milliseconds to delay
 /// @param run_async If true, pump async tasks during delay (only on platforms with SKETCH_HAS_LARGE_MEMORY==1)
-void delay(u32 ms, bool run_async = true) FL_NOEXCEPT;
+void delay_impl(u32 ms, bool run_async = true) FL_NOEXCEPT;
 
-/// Template overload for all arithmetic types - coexists with Arduino's extern "C" delay()
-/// Templates have different overload resolution rules and don't conflict with C-linkage functions
-/// This allows "using fl::delay;" to work universally on all platforms including Arduino
-/// @param ms Milliseconds to delay (accepts integers, floats, doubles)
-/// @note Similar technique used by fl::round to coexist with Arduino's ::round()
-/// @note Accepts all integer types (char, short, int, long, etc.) and floating-point types (float, double)
-template<typename T, typename = fl::enable_if_t<fl::is_arithmetic<T>::value>>
-inline void delay(T ms) FL_NOEXCEPT {
-    delay(static_cast<u32>(ms), true);
-}
+}  // namespace detail
 
-/// Template overload with async flag for all arithmetic types
-/// @param ms Milliseconds to delay (accepts integers, floats, doubles)
-/// @param run_async If true, pump async tasks during delay
-/// @note Accepts all integer types (char, short, int, long, etc.) and floating-point types (float, double)
-template<typename T, typename = fl::enable_if_t<fl::is_arithmetic<T>::value>>
-inline void delay(T ms, bool run_async) FL_NOEXCEPT {
-    delay(static_cast<u32>(ms), run_async);
+/// Public delay wrapper that keeps bare Arduino delay() preferred after
+/// `using fl::delay;` while still allowing explicit fl::delay() calls.
+/// @param ms Milliseconds to delay
+/// @param run_async If true, pump async tasks during delay (only on platforms with SKETCH_HAS_LARGE_MEMORY==1)
+template<int Dummy = 0>
+inline void delay(u32 ms, bool run_async = true) FL_NOEXCEPT {
+    (void)Dummy;
+    detail::delay_impl(ms, run_async);
 }
 
 /// Delay for a given number of milliseconds (legacy - no async pumping)
@@ -124,7 +118,7 @@ inline void delayUs(u32 us) FL_NOEXCEPT {
 /// @param ms Milliseconds to delay
 /// @param run_async If true, pump async tasks during delay (only on platforms with SKETCH_HAS_LARGE_MEMORY==1)
 inline void delayMs(u32 ms, bool run_async = true) FL_NOEXCEPT {
-  delay(ms, run_async);
+  detail::delay_impl(ms, run_async);
 }
 
 /// Shorter alias for delayNanoseconds (template version)

--- a/tests/fl/system/delay.cpp
+++ b/tests/fl/system/delay.cpp
@@ -7,6 +7,18 @@
 
 FL_TEST_FILE(FL_FILEPATH) {
 
+namespace delay_lookup_regression {
+char delay(fl::u32);
+using fl::delay;
+
+static_assert(fl::is_same<decltype(delay(fl::u32{1})), char>::value,
+              "Bare delay(u32) should prefer the non-template Arduino-style overload");
+static_assert(fl::is_same<decltype(delay(static_cast<unsigned long>(1))), char>::value,
+              "Bare delay(unsigned long) should prefer the non-template Arduino-style overload");
+static_assert(fl::is_same<decltype(delay(fl::u32{1}, false)), void>::value,
+              "Two-argument delay(ms, run_async) should resolve to fl::delay");
+}  // namespace delay_lookup_regression
+
 // ============================================================================
 // Test Suite: Compile-time Template Delays
 // ============================================================================

